### ISSUE TITLE
Some VPopovers are missing labels

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1012]
## Description

Some of the Popovers are missing labels and log warnings about it in the console: 
https://github.com/WordPress/openverse-frontend/blob/3ab03476a378768d329e3a21ec0f3a402bb4b560/src/components/VContentReport/VContentReportPopover.vue#L2-L6

https://github.com/WordPress/openverse-frontend/blob/3ab03476a378768d329e3a21ec0f3a402bb4b560/src/components/VFilters/VFilterChecklist.vue#L25

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.
